.travis.yml

```diff
@@ -4,10 +4,21 @@ node_js:
 - '6'
 - '8'
 script:
-  - yarn run test-ci
+  # - yarn run test-ci
   - node bin/build-locales
   - yarn run build
   - yarn run lint
+
+  # run addons-linter binary in a yarn production environment, clean parent
+  # environment first though.
+  - rm -rf ./node_modules
```

Comment:
this is fine for now, but we should use the same strategy (run on a "production like environment") also for the upcoming integration tests, and in that case we should probably just move the directory where we unpack and install the release version into another top level dir (e.g. creating it in a /tmp subdir on travis could be ok), so that we can still use jest (that is in our dev dependencies) to run the test suite.

Reply:
Yeah, I wasn't so sure about the `/tmp` directory as travis does have weird filesystem rules and before I run into debugging travis itself I took the safe-path :see_no_evil: 

I'll file follow-up issues.